s3.tf

```diff
@@ -1,7 +1,41 @@
 resource "aws_s3_bucket" "demo-example" {
```

Comment:
Accurics remediation:
Title: HIGH: S3 bucket accessible to all authenticated users.
Description:Vulnerability: S3 bucket Access is allowed to all AWS Account Users./n/nRemediation:S3 bucket Access should be limited to specific users and roles.